schema/crdb/35.0.0/up.sql

```diff
@@ -0,0 +1,6 @@
+-- This isn't realy a schema migration, but is instead a one-off fix for
+-- incorrect data (https://github.com/oxidecomputer/omicron/issues/5056) after
+-- the root cause for the incorrect data has been addressed.
+UPDATE omicron.public.network_interface
```

Comment:
Just checking, but are there any situations in which we'd have more than one NIC per parent? I believe we'd trip the unique index you set up in the last PR, in any case.

Reply:
I agree we'd trip on the index, which would be pretty painful during an attempted schema migration. However, I don't think it's possible; RSS assigns exactly one vNIC to each Nexus, External DNS, and Boundary NTP zone, and those are the only service vNICs ever created (at least until we start adding more as part of reconfiguration).